projects/test-app/src/hslayers-app/hslayers-app.component.html

```diff
@@ -1,50 +1,3 @@
-                tincidunt id aliquet risus. Nisl rhoncus mattis rhoncus urna neque viverra justo.</p>
-
-            <p>Sit amet porttitor eget dolor morbi non arcu risus. Arcu dui vivamus arcu felis bibendum ut tristique et.
-                Eget egestas purus viverra accumsan in. Dignissim convallis aenean et tortor at risus viverra adipiscing
-                at. Ultrices eros in cursus turpis massa tincidunt dui ut. Vitae sapien pellentesque habitant morbi
-                tristique senectus et netus. Et sollicitudin ac orci phasellus egestas tellus rutrum tellus. Sem et
-                tortor consequat id porta nibh venenatis cras. Et pharetra pharetra massa massa ultricies mi. Morbi quis
-                commodo odio aenean sed adipiscing diam donec adipiscing. Semper auctor neque vitae tempus quam. Enim
-                blandit volutpat maecenas volutpat blandit aliquam etiam erat. Lacus viverra vitae congue eu.</p>
-
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-                    dolore magna aliqua. Scelerisque viverra mauris in aliquam sem fringilla ut morbi. Amet dictum sit amet
-                    justo donec enim diam. Nisi est sit amet facilisis magna etiam. Sit amet luctus venenatis lectus magna
-                    fringilla urna. Dapibus ultrices in iaculis nunc sed augue lacus viverra vitae. Semper eget duis at
-                    tellus at urna condimentum. Ut tristique et egestas quis ipsum suspendisse. Ullamcorper morbi tincidunt
-                    ornare massa eget egestas purus. Suspendisse faucibus interdum posuere lorem ipsum dolor. Lacus luctus
-                    accumsan tortor posuere ac ut consequat semper viverra. Accumsan sit amet nulla facilisi. Sed turpis
-                    tincidunt id aliquet risus. Nisl rhoncus mattis rhoncus urna neque viverra justo.</p>
-    
-                <p>Sit amet porttitor eget dolor morbi non arcu risus. Arcu dui vivamus arcu felis bibendum ut tristique et.
-                    Eget egestas purus viverra accumsan in. Dignissim convallis aenean et tortor at risus viverra adipiscing
-                    at. Ultrices eros in cursus turpis massa tincidunt dui ut. Vitae sapien pellentesque habitant morbi
-                    tristique senectus et netus. Et sollicitudin ac orci phasellus egestas tellus rutrum tellus. Sem et
-                    tortor consequat id porta nibh venenatis cras. Et pharetra pharetra massa massa ultricies mi. Morbi quis
-                    commodo odio aenean sed adipiscing diam donec adipiscing. Semper auctor neque vitae tempus quam. Enim
-                    blandit volutpat maecenas volutpat blandit aliquam etiam erat. Lacus viverra vitae congue eu.</p>
-        </div>
-        <div class="col col-1">
-
-        </div>
-    </div>
-</div>
-<hslayers *ngIf="multiAppMode" style="height: 70vh" id="app-2"></hslayers>
+<hslayers></hslayers>
```

Comment:
Two hslayers elements here would result in 4 maps because we are multiplying one layer up e.g. hslayers-app element.
And actually, I feel like its not necessary anymore because for 'full-app' to get two maps with different configs you just need to create another module and that's just same thing done twice.
 So something like this would have to be done in index.html which however is outside of angular so no dricetives etc.